sys/include/net/nanocoap.h

```diff
@@ -973,6 +973,22 @@ static inline ssize_t coap_opt_add_format(coap_pkt_t *pkt, uint16_t format)
  */
 ssize_t coap_opt_add_opaque(coap_pkt_t *pkt, uint16_t optnum, const uint8_t *val, size_t val_len);
 
+/**
+ * @brief   Adds one or multiple Uri-Query options in the form '/path' into pkt
+ *
+ * @note Use this only for null-terminated strings.
+ *
+ * @param[in,out] pkt         Packet being built
+ * @param[in]     path        Resource (sub)path
+ *
+ * @pre     ((pkt != NULL) && (path != NULL))
+ *
+ * @return        number of bytes written to pkt buffer
+ * @return        <0 on error
+ * @return        -ENOSPC if no available options or pkt full
+ */
+ssize_t coap_opt_add_path(coap_pkt_t *pkt, const char *path);
```

Comment:
Please rename to `coap_opt_add_uri_path()` for consistency with other functions. The principle is to use the canonical name for the option in the function name. There also is a Location-Path option. And yes, yes, I know `coap_opt_add_uquery()` violates this rule. I plan to fix that next. ;-)